lib/zipkin-tracer/faraday/zipkin-tracer.rb

```diff
@@ -29,14 +27,21 @@ def call(env)
 
     SERVER_ADDRESS_SPECIAL_VALUE = '1'.freeze
 
-    def b3_headers
-      {
-        trace_id: 'X-B3-TraceId',
-        parent_id: 'X-B3-ParentSpanId',
-        span_id: 'X-B3-SpanId',
-        sampled: 'X-B3-Sampled',
-        flags: 'X-B3-Flags'
-      }
+    B3_HEADERS = {
```

Comment:
The changes are good but can we keep this method as it is instead? 
We have a gem internally that depens on it.

Otherwise I'd merge the pr